middleware/favicon/README.md

```diff
@@ -42,6 +42,7 @@ app.Use(favicon.New())
 ```go
 app.Use(favicon.New(favicon.Config{
 	File: "./favicon.ico",
+	URL: "/favicon.ico"
```

Comment:
Please format, i.e. add another space character between the colon and the opening quote.